change/@fluentui-react-native-tester-83c37bb6-6aa1-4b48-866a-d72414a031f4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Remove janky scroll logic for win32 component button",
```

Comment:
This term is a bit vague to me. I would be more specific and say something like say "Remove unnecessary scroll logic..." or something to that effect.